shaptools/hdb_connector/connectors/dbapi_connector.py

```diff
@@ -16,6 +16,18 @@
 from shaptools.hdb_connector.connectors import base_connector
 
 
+class DbapiQueryResult(base_connector.QueryResult):
```

Comment:
Some inheritance laws are broken here. The parameters in the parent and inherited classes are different. Even though Python accepts that (it almost accepts everything...) we shouldn't do that.

Anyway, I don't really think that we need more subclasses here. Having the main one is enough in my opinion